.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,47 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the Tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Clone the crud module
+        uses: actions/checkout@v3
+        with:
+          repository: ${{ github.repository_owner }}/crud
```

Comment:
What's this one for?

Reply:
This one to checkout crud module like @ylobankov did at [expirationd](https://github.com/tarantool/expirationd/blob/3bbc7fdb9f53d1de01457a07a093f313e2752135/.github/workflows/reusable_testing.yml#L19)

Reply:
Is it a v3 thing? There is no any additional args with v2 checkout

Reply:
It is a limitation of reusable workflows. Reusable workflows don't have the own contexts, so if you call this workflow inside `tarantool/tarantool`, this step will check out the `tarantool/tarantool` repo instead of `tarantool/crud`. That's why we need the repository here.

Reply:
From GitHub [docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows#overview):

>If you reuse a workflow from a different repository, any actions in the called workflow run as if they were part of the caller workflow. For example, if the called workflow uses `actions/checkout`, the action checks out the contents of the repository that hosts the caller workflow, not the called workflow.

>When a reusable workflow is triggered by a caller workflow, the `github` context is always associated with the caller workflow. The called workflow is automatically granted access to `github.token` and `secrets.GITHUB_TOKEN`. 
```